stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/nativeimpl/UnregisterTopicAtHub.java

```diff
@@ -46,7 +47,12 @@
     public void execute(Context context) {
         String topic = context.getStringArgument(0);
         String secret = context.getStringArgument(1);
-        context.setReturnValues(new BString(Hub.getInstance().unregisterTopic(topic, secret)));
+        try {
+            Hub.getInstance().unregisterTopic(topic, secret);
+            context.setReturnValues();
+        } catch (BallerinaWebSubException e) {
+            BLangVMErrors.createError(context, e.getMessage());
```

Comment:
Error is not returned here. Need to add setReturnValues ?

Reply:
Yes, thanks! fixed with 66d27cc.